[FIX] change remaining latin expressions (etc and i.e.) (#628)
bids-standard/bids-specification: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

| **Key name**                | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                   |
| --------------------------- | --------------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | PulseSequenceType           | RECOMMENDED           | [string][]                             | A general description of the pulse sequence used for the scan (for example: MPRAGE, Gradient Echo EPI, Spin Echo EPI, Multiband gradient echo EPI).                                                                                                                       |

Reviewer: ```suggestion
| PulseSequenceType           | RECOMMENDED           | [string][]                             | A general description of the pulse sequence used for the scan (for example, MPRAGE, Gradient Echo EPI, Spin Echo EPI, Multiband gradient echo EPI).                                                                                                                       |
```

manipulations). See [here](https://lcni.uoregon.edu/kb-articles/kb-0003) and
[here](https://github.com/neurolabusc/dcm_qa/tree/master/In/TotalReadoutTime)

> <sup>3</sup>We use the "FSL definition", that is the time between the center of the
> first "effective" echo and the center of the last "effective" echo.

Reviewer: ```suggestion
<sup>3</sup>We use the time between the center of the first "effective" echo
and the center of the last "effective" echo, sometimes called the "FSL definition".
```

| InversionTime          | RECOMMENDED                                                                                                      | [number][]               | The inversion time (TI) for the acquisition, specified in seconds. Inversion time is the time after the middle of inverting RF pulse to middle of excitation pulse to detect the amount of longitudinal magnetization. Corresponds to DICOM Tag 0018, 0082 `Inversion Time` (please note that the DICOM term is in milliseconds not seconds).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

> | SliceTiming            | RECOMMENDED, but REQUIRED for sparse sequences that do not have the `DelayTime` field set                        | [array][] of [numbers][] | The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis (that is that the final entry in the `SliceTiming` list is the time of acquisition of slice 0). Without this parameter slice time correction will not be possible. |

Reviewer: I think this is clearer as a subordinate clause. I think we probably overuse parentheses.

```suggestion
| SliceTiming            | RECOMMENDED, but REQUIRED for sparse sequences that do not have the `DelayTime` field set                        | [array][] of [numbers][] | The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis, suh that the final entry in the `SliceTiming` list is the time of acquisition of slice 0. Without this parameter slice time correction will not be possible. |
```

Author: I guess you meant "so" or that's an attempt at rendering some local accent I am not familiar with. :smile:

Reviewer: Yup, "such". Updated the suggestion.

| InversionTime          | RECOMMENDED                                                                                                      | [number][]               | The inversion time (TI) for the acquisition, specified in seconds. Inversion time is the time after the middle of inverting RF pulse to middle of excitation pulse to detect the amount of longitudinal magnetization. Corresponds to DICOM Tag 0018, 0082 `Inversion Time` (please note that the DICOM term is in milliseconds not seconds).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

> | SliceTiming            | RECOMMENDED, but REQUIRED for sparse sequences that do not have the `DelayTime` field set                        | [array][] of [numbers][] | The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis (meaning that the final entry in the `SliceTiming` list is the time of acquisition of slice 0). Without this parameter slice time correction will not be possible. |

Reviewer: that is,

> | SliceTiming            | RECOMMENDED, but REQUIRED for sparse sequences that do not have the `DelayTime` field set                        | [array][] of [numbers][] | The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis (meaning that the final entry in the `SliceTiming` list is the time of acquisition of slice 0). Without this parameter slice time correction will not be possible. |

Author: ```suggestion
| SliceTiming            | RECOMMENDED, but REQUIRED for sparse sequences that do not have the `DelayTime` field set                        | [array][] of [numbers][] | The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `SliceTiming` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `SliceEncodingDirection` is negative, the entries in `SliceTiming` are defined in reverse order with respect to the slice axis (that is that the final entry in the `SliceTiming` list is the time of acquisition of slice 0). Without this parameter slice time correction will not be possible. |
```

        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_sbref.nii[.gz]
```

> Imaging data acquired during functional imaging (referring to imaging which supports

Reviewer: that is,

> Imaging data acquired during functional imaging (referring to imaging which supports

Author: ```suggestion
Imaging data acquired during functional imaging (that is, to imaging which supports
```

rapid temporal repetition). This includes but is not limited to task based fMRI

> as well as resting state fMRI (meaning that rest is treated as another task). For task

Reviewer: ```suggestion
as well as resting state fMRI, which is treated like any other task. For task
```

        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_ce-<label>][_dir-<label>][_rec-<label>][_run-<index>][_echo-<index>]_sbref.nii[.gz]
```

> Imaging data acquired during functional imaging (that is, to imaging which supports
> rapid temporal repetition). This includes but is not limited to task based fMRI

Reviewer: This is a sentence fragment, so we might as well fix that while we're here. What about:

```suggestion
Functional imaging consists of techniques that support rapid temporal repetition.
This includes but is not limited to task based fMRI
```

diffusion gradient, where the i-th elements in each row correspond together to
the i-th volume with `[0,0,0]` for non-diffusion-weighted volumes. Inherent to
the FSL format for bvec specification is the fact that the coordinate system of

> the bvecs is with respect to the participant (meaning defined by the axes of the

Reviewer: that is,

> the bvecs is with respect to the participant (meaning defined by the axes of the

Author: ```suggestion
the bvecs is with respect to the participant (that is defined by the axes of the
```

diffusion gradient, where the i-th elements in each row correspond together to
the i-th volume with `[0,0,0]` for non-diffusion-weighted volumes. Inherent to
the FSL format for bvec specification is the fact that the coordinate system of

> the bvecs is with respect to the participant (that is defined by the axes of the
> corresponding dwi.nii file) and not the magnet's coordinate system, which means
> that any rotations applied to dwi.nii also need to be applied to the
> corresponding bvec file.

Reviewer: ```suggestion
the bvecs is with respect to the participant, defined by the axes of the
corresponding `dwi.nii` file, and not the magnet's coordinate system.
Thus, any rotations applied to `dwi.nii` also need to be applied to the
corresponding bvec file.
```
